internal/pkg/consensus/processor.go

```diff
@@ -64,28 +41,25 @@ type ApplyMessageResult struct {
 
 // DefaultProcessor handles all block processing.
 type DefaultProcessor struct {
-	validator     MessageValidator
-	blockRewarder BlockRewarder
-	actors        builtin.Actors
+	validator MessageValidator
```

Comment:
I don't see this getting called anywhere, is the validation done elsewhere in the new vm runtime?  If not I think this is a bug and we should introduce a call here.  If so I think this dependency can be deleted.

Reply:
I'll delete it for now